cmd/account/cli.go

```diff
@@ -102,26 +102,27 @@ func (o *cliOptions) run() error {
 	}
 
 	if o.output == "" {
-		fmt.Fprintf(o.IOStreams.Out, "Temporary AWS Credentials:\n%s\n", creds)
+		fmt.Println("Temporary AWS Credentials:")
 	}
 
-	if o.output == "json" {
-		fmt.Fprintf(o.IOStreams.Out, "{\n\"AccessKeyId\": %q, \n\"Expiration\": %q, \n\"SecretAccessKey\": %q, \n\"SessionToken\": %q\n}",
+	if o.output == "env" || o.output == "" {
+		fmt.Fprintf(o.IOStreams.Out, "AWS_ACCESS_KEY_ID=%s \nAWS_SECRET_ACCESS_KEY=%s \nAWS_SESSION_TOKEN=%s \nAWS_DEFAULT_REGION=%s\n",
 			*creds.AccessKeyId,
-			*creds.Expiration,
 			*creds.SecretAccessKey,
 			*creds.SessionToken,
+			o.k8sclusterresourcefactory.Awscloudfactory.Region,
 		)
 	}
 
-	if o.output == "env" {
-		fmt.Fprintf(o.IOStreams.Out, "AWS_ACCESS_KEY_ID=%s \nAWS_SECRET_ACCESS_KEY=%s \nAWS_SESSION_TOKEN=%s \nAWS_DEFAULT_REGION=%s\n",
+	if o.output == "json" {
+		fmt.Fprintf(o.IOStreams.Out, "{\n\"AccessKeyId\": %q, \n\"Expiration\": %q, \n\"SecretAccessKey\": %q, \n\"SessionToken\": %q\n}",
 			*creds.AccessKeyId,
+			*creds.Expiration,
 			*creds.SecretAccessKey,
 			*creds.SessionToken,
-			o.k8sclusterresourcefactory.Awscloudfactory.Region,
 		)
 	}
 
+
```

Comment:
Looks like the lint step is complaining about this extra line.

Reply:
thanks... fixed